packages/framework-provider-azure-infrastructure/src/infrastructure/constants.ts

```diff
@@ -1,2 +1,2 @@
-export const MAX_CONTAINER_THROUGHPUT = 4000
-export const MAX_DATABASE_THROUGHPUT = 4000
+export const MAX_CONTAINER_THROUGHPUT = 400
+export const MAX_DATABASE_THROUGHPUT = 400
```

Comment:
Hi @heavylemon1, first of all, thanks for contributing, it really helps!

These parameters are a bit sensible, and it would be a good idea to allow tweaking them. As a quick-n-dirty solution, we could just try loading them from environment variables like this:

```suggestion
export const MAX_CONTAINER_THROUGHPUT = process.env.AZURE_MAX_CONTAINER_THROUGHPUT ?? 400
export const MAX_DATABASE_THROUGHPUT = process.env.AZURE_MAX_DATABASE_THROUGHPUT ?? 400
```

If it makes sense to you, could you please add a note to the documentation to make these configuration options visible? I'd say this is the right place: https://docs.boosterframework.com/going-deeper/infrastructure-providers/#azure-provider-configuration

You can find the documentation in the `website` folder in the same Booster repository.

Thanks!

Reply:
Got it @javiertoledo , I will be changing those as suggested and was also thinking of maybe using `parseInt` since CosmosDB Throughput is based on read units which are integers from my understanding (so to ensure they will be as expected).

So, I will first change the implementation to process env,
update the documentation and
run `rush change`.

Thanks!